recipe/conda_build_config.yaml

```diff
@@ -2,3 +2,7 @@ c_stdlib_version:  # [unix]
   - '2.28'  # [linux and x86_64]
 docker_image:
   - quay.io/condaforge/linux-anvil-alma-x86_64:8  # [linux and x86_64]
+channel_sources:  # [win]
+  - conda-forge/label/m2w64-experimental,conda-forge  # [win]
+channel_targets:  # [win]
+  - conda-forge m2w64-experimental  # [win]
```

Comment:
I am not sure whether the package needs this `channel_targets` - is the package incompatible with other Windows packages that are not compiled with `m2w64-experimental`? If so, it is basically useless; if not, then we can put it in the main conda-forge channel, can't we?

/cc @xhochy @traversaro

Reply:
I think it is fine to be put on main. The special channel is only needed to pull the newer `m2w64` compilers.